tests/databases/molecule/test_get_all.py

```diff
@@ -40,56 +53,43 @@ def test_get_all(mongo_client):
         stk.BuildingBlock('NCCN'),
         stk.BuildingBlock('CCCCC'),
         stk.BuildingBlock('NCCCCN'),
+        stk.BuildingBlock('NCC(CCBr)CCN'),
+        stk.BuildingBlock('NCCNCCC(CCBr)CCN'),
     )
 
-    # Two sets of molecules with two distinct molecules each, and one
-    # shared molecule to test for overlap.
-    molecules1 = (all_molecules[0], all_molecules[1], all_molecules[4])
-    molecules2 = (all_molecules[2], all_molecules[3], all_molecules[4])
+    inchi_molecules = all_molecules[:3]
+    smiles_molecules = all_molecules[3:-2]
+    inchi_and_smiles_molecules = all_molecules[-2:]
 
-    for molecule in molecules1:
-        database1.put(molecule)
+    for molecule in inchi_molecules:
+        inchi_database.put(molecule)
 
-    for molecule in molecules2:
-        database2.put(molecule)
+    for molecule in smiles_molecules:
+        smiles_database.put(molecule)
 
-    # A new database with both indices.
-    key_makers = (stk.Inchi(), stk.Smiles())
-    jsonizer = stk.MoleculeJsonizer(key_makers=key_makers)
-    database3 = stk.MoleculeMongoDb(
-        mongo_client=mongo_client,
-        database=database_name,
-        jsonizer=jsonizer,
-        put_lru_cache_size=0,
-        get_lru_cache_size=0,
-        indices=(i.get_key_name() for i in key_makers),
-    )
+    for molecule in inchi_and_smiles_molecules:
+        inchi_and_smiles_database.put(molecule)
 
-    molecules1_by_key = {
-        key_makers[0].get_key(molecule): molecule
-        for molecule in molecules1
-    }
-    molecules2_by_key = {
-        key_makers[1].get_key(molecule): molecule
-        for molecule in molecules2
+    smiles_to_molecule = {
```

Comment:
why no inch_to_molecule?

Reply:
it would serve no purpose, the point of this dict is to find the expected molecule. you only need a smiles for that. (any key maker that works with any moleulce would do really)